services/crud_helper.py

```diff
@@ -60,7 +73,17 @@ def model_patcher(
     """
 
     for key, value in payload.model_dump(exclude_unset=True).items():
-        setattr(item, key, value)
+        if isinstance(item, NotesMixin) and key == "notes":
```

Comment:
The `NotesMixin` is to add the notes attribute to a table. Should this be `if issubclass(table, Notes)`?